src/type/preload.ts

```diff
@@ -255,10 +255,19 @@ export type SavingSetting = {
   avoidOverwrite: boolean;
   exportText: boolean;
   outputStereo: boolean;
-  outputSamplingRate: number | "engineDefault";
   audioOutputDevice: string;
 };
 
+export type EngineSetting = Record<string, EngineSettingRecord>;
+
+export const engineSettingRecord = z.object({
+  useGpu: z.boolean().default(false),
+  outputSamplingRate: z
+    .union([z.number(), z.literal("engineDefault")])
+    .default("engineDefault"),
+});
+export type EngineSettingRecord = z.infer<typeof engineSettingRecord>;
```

Comment:
ここ、型名のRecordと型のRecordが一致してなさそうです。
こう？
```suggestion
export type EngineSettingRecord = Record<string, EngineSetting>;

export const engineSetting = z.object({
  useGpu: z.boolean().default(false),
  outputSamplingRate: z
    .union([z.number(), z.literal("engineDefault")])
    .default("engineDefault"),
});
export type EngineSetting = z.infer<typeof engineSetting>;
```